app/views/papers/show.json.jbuilder

```diff
@@ -0,0 +1,20 @@
+json.title @paper.title
+json.state @paper.state
+json.submitted_at @paper.created_at
+if @paper.published?
+  json.doi @paper.doi
+  json.published_at @paper.accepted_at
+  json.volume @paper.volume
+  json.issue @paper.issue
+  json.year @paper.year
+  json.page @paper.page
+  json.authors @paper.metadata_authors
+  json.editor @paper.metadata_editor
+  json.reviewers @paper.metadata_reviewers
+  json.languages @paper.language_tags.join(', ')
+  json.tags @paper.author_tags.join(', ')
+  json.software_repository @paper.repository_url
+  json.paper_review @paper.review_url
+  json.pdf_url @paper.pdf_url
```

Comment:
```suggestion
  json.pdf_url @paper.seo_pdf_url
```

Let's switch this to the URL that we use for Google Scholar rather than the GitHub Pages one.